templatefinder/tests.py

```diff
@@ -1,5 +1,8 @@
 from django.conf import settings
-from django.utils import unittest
+try:
+    from django.utils import unittest
+except ImportError:
+    import unittest
```

Comment:
Can you just add a comment explaining when does the `ImportError` happen and why we need to do it that way?